docs/running-with-FIPS.md

```diff
@@ -1,102 +1,143 @@
-# Using the Unix Tunneler in FIPS mode.
+# FIPS 140-3 Compliance
 
-This guide applies to Linux and Darwin (macOS).
+[FIPS](https://www.nist.gov/itl/fips-general-information) (Federal Information Processing Standards) are security
+standards developed by NIST (National Institute of Standards and Technology), a U.S. government agency responsible for
+promoting innovation and industrial competitiveness through standards and technology.
 
-## Overview
+This software is built to comply with the FIPS 140-3 program, specifically with OpenSSL as of May 2025. The software is
+considered "FIPS 140-3 compliant", it is **not FIPS validated**. Per the FAQ found at
+https://csrc.nist.gov/Projects/cryptographic-module-validation-program/faqs:
 
-1. Install a release package or build `ziti-edge-tunnel` from source
-2. Install the OpenSSL FIPS module
-3. Configure the OpenSSL FIPS module
-4. Run the Tunneler
+> FIPS 140-3 "compliant" means a vendor believes its product implementation meets the FIPS 140-3 requirements,
+> but the product has not gone through the CMVP validation process.
 
-## Acquiring the Tunneler Binary
+## Enabling FIPS Compliance
 
-You may install a release package on Debian or RedHat Linux, or build `ziti-edge-tunnel` from source.
+FIPS-compliance can be initiated in one of two ways: environment variable or by placing an `openssl.cnf` file adjacent to
+the ziti-edge-tunnel executable. Both options will trigger the statically linked OpenSSL code within the ziti-edge-tunnel
```

Comment:
The ZET shipped for compliant-ready environments like Ubuntu 22.04 and RedHat 9 are not statically linked, so this guidance may be specific to Windows.

Reply:
sounds good